rust/gui-client/src-tauri/src/settings/windows.rs

```diff
@@ -0,0 +1,32 @@
+use super::MdmSettings;
+use anyhow::Result;
+
+pub fn load_mdm_settings() -> Result<MdmSettings> {
+    let registry_values = MdmRegistryValues::load_from_registry()?;
+
+    Ok(MdmSettings {
+        auth_url: registry_values.authURL.and_then(|url| url.parse().ok()),
+        api_url: registry_values.apiURL.and_then(|url| url.parse().ok()),
+        log_filter: registry_values.logFilter,
+        account_slug: registry_values.accountSlug,
+        hide_admin_portal_menu_item: registry_values.hideAdminPortalMenuItem,
+        connect_on_start: registry_values.connectOnStart,
+        check_for_updates: registry_values.checkForUpdates,
+        support_url: registry_values.supportURL.and_then(|url| url.parse().ok()),
+    })
+}
+
+/// Windows-specific struct for ADMX-backed MDM settings.
+#[derive(Clone, Debug)]
+#[admx_macro::admx(path = "../website/public/policy-templates/windows/firezone.admx")]
```

Comment:
@jamilbk The latest version of this branch now uses a procedural macro to parse the ADMX file and map the defined registry keys to these struct fields. This means we will get a compile error whenever we update the ADMX file without also updating this struct here. It also automatically reads the correct registry key.

This should make it significantly easier to iterate on this config and add more values later without having to perform a full e2e test every time.